configuration/micro_kernel_trait.rst

```diff
@@ -128,12 +128,17 @@ your ``composer.json`` file to load from there:
     }
 
 Now, suppose you want to use Twig and load routes via annotations. For annotation
-routing, you need SensioFrameworkExtraBundle. This comes with a normal Symfony project.
+routing, you need the Doctrine annotations parser. This comes with a normal Symfony project.
 But in this case, you need to download it:
 
 .. code-block:: bash
 
-    $ composer require sensio/framework-extra-bundle
+    $ composer require doctrine/annotations
```

Comment:
Since the article requires symfony/symfony above, will doctrine annotations already be downloaded? I wasn't sure